src/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,152 @@
+.navbar {
+  &__logo {
+    width: 100px;
+  }
+}
+.hamburger .line {
+  width: 50px;
+  height: 5px;
+  background-color: #243b70;
+  display: block;
+  margin: 4px 4px 4px 0;
+  -webkit-transition: all 0.3s ease-in-out;
+  -o-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.hamburger:hover {
+  cursor: pointer;
+}
+
+#hamburger {
```

Comment:
I think that even for the mobile the whole `Navbar` shouldn't disappear. The only change should generate a `Navbar` with the logo on the left and with the hamburger icon (button) on the right, which opens and close the mobile menu.